bundles/org.openhab.binding.jellyfin/src/main/java/org/openhab/binding/jellyfin/internal/handler/JellyfinClientHandler.java

```diff
@@ -183,6 +215,14 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         }
     }
 
+    @NotNull
```

Comment:
Related to my other comment

Reply:
Seems like I don't need it at all, I think it got added when I generated the method with the refactor into method functionality of the IDE. Thanks for pointing it.